File: Service/Implement/DuAnQuyetToanService.cs
namespace Service.Implement
{
    public class DuAnQuyetToanService : BaseService<DuAnQuyetToan, IDuAnQuyetToanRepository>
    , IDuAnQuyetToanService
    {
    private readonly IDuAnQuyetToanRepository _DuAnQuyetToanRepository;
    public DuAnQuyetToanService(IDuAnQuyetToanRepository DuAnQuyetToanRepository) : base(DuAnQuyetToanRepository)
    {
    _DuAnQuyetToanRepository = DuAnQuyetToanRepository;
    }
    }
    }

File: CapSo/src/app/shared/DuAnQuyetToan.service.ts
import { Injectable } from '@angular/core';
import { HttpClient, HttpParams } from '@angular/common/http';
import { environment } from 'src/environments/environment';
import { DuAnQuyetToan } from 'src/app/shared/DuAnQuyetToan.model';
import { BaseService } from './Base.service';
@Injectable({
    providedIn: 'root'
})
export class DuAnQuyetToanService extends BaseService{
    DisplayColumns001: string[] = ['Save', 'STT', 'ID', 'ParentID', 'ParentName', 'CreatedDate', 'CreatedMembershipID', 'LastUpdatedDate', 'LastUpdatedMembershipID', 'RowVersion', 'SortOrder', 'Active', 'TypeName', 'Name', 'Code', 'Note', 'Display', 'FileName', 'Description', 'HTMLContent', 'DanhMucNgonNguID', 'DanhMucUngDungID', 'DanhMucNgonNguName', 'DanhMucUngDungName', 'Nam', 'Quy', 'Thang', 'Ngay', 'GhiCo', 'GhiNo', 'ConLai'];
    constructor(public httpClient: HttpClient) {
        super(httpClient);
        this.Controller = "DuAnQuyetToan";
    }
}

